services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/liitteet/AttachmentServiceImpl.kt

```diff
@@ -25,35 +24,57 @@ class AttachmentServiceImpl(
     val hankeRepository: HankeRepository,
     val attachmentRepository: AttachmentRepository,
 ) : AttachmentService {
+
+    fun getExtension(filename: String): String {
```

Comment:
This can be replaced at the call site with `FilenameUtils.getExtension(filename)` from Apache Commons.